travis-ci/travis-web
# Show discount on subscription tab

Ticket: https://github.com/travis-pro/billing-v2/issues/409

Steps to test manually:
- Create a subscription with a coupon.
- On the subscription tab, you should see the coupon validity on the dashboard.
